docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '2.4'
+
+services:
+  jekyll:
+    image: bretfisher/jekyll-serve
+    volumes:
+      - .:/site
+    ports:
+      - '4000:4000'
```

Comment:
I'm not against adding that if you find it helpful, but some doc in the README on how to use it would enable other people to use it.